libraries/AP_GPS/AP_GPS.h

```diff
@@ -72,7 +72,8 @@ class AP_GPS
         GPS_TYPE_QURT  = 12,
         GPS_TYPE_ERB = 13,
         GPS_TYPE_MAV = 14,
-        GPS_TYPE_NOVA = 15,
+		GPS_TYPE_NOVA = 15,
```

Comment:
4 spaces rather then tab is the standard. http://ardupilot.org/dev/docs/style-guide.html#spaces